book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals << rental
```

Comment:
- [ ] The `rentals` here should be the instance variable.

  ```
  @rentals << rental
  ```

Reply:
done